CHANGES.adoc

```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+== 0.0.35 Jan 01 2020
+
+- Fixes for `CurrentAccess` resource.
+
```

Comment:
Do you mind committing this file in a seperate commit? Its custome in this repo that changes to `CHANGES.adoc` are committed in a separate commit titles `Release x.x.x`.